circle.yml

```diff
@@ -13,4 +13,7 @@ dependencies:
     - "vendor"
 test:
   override:
-    - vendor/bin/phpunit --bootstrap ./tests/bootstrap.php tests
\ No newline at end of file
+    - vendor/bin/phpunit --bootstrap ./tests/bootstrap.php tests
+    
+    
+    # demo buld on github
```

Comment:
Fix typo: `demo build on GitHub`.

Reply:
Hello, thanks for this contribution. I would actually prefer to keep the line break you did, but without line 18 and 19. Could you do that? Thanks!